pydis_site/templates/events/pages/code-jams/the-code-style-guide.html

```diff
@@ -0,0 +1,257 @@
+        but what if the project has thousands and thousands of files in a really complex folder structure?
+        Do you want then code what looks like the first example?
+        You can save hours sometimes if you write beautiful code that follows style rules.
+    </p>
+    <p>
+        The most important document of Python code style is <b><a href="https://www.python.org/dev/peps/pep-0008/">PEP 8</a></b>.
+        This gives the majority of all Python code style rules. This article will cover the most important rules of PEP 8.
+    </p>
+
+    <h2>Linters</h2>
+    <p>
+        But everyone makes mistakes and there are so many style rules that can be really difficult to remember and always follow.
+        Luckily, we have amazing tools that help us - linters. While there are many linters,
+        we'd like code jam participants to use <b><a href="https://flake8.pycqa.org/en/latest/">flake8</a></b>.
+        Flake8 points out to you rules what you did break in your code so you can fix them.
+    </p>
+
+    <h2>Rules</h2>
+    <h3>Basics</h3>
+    <p>For indentation, you should use 4 spaces. Using tabs is not suggested, but if you do, you can't mix spaces and tabs.</p>
+    <p>
+        PEP 8 defines a maximum line length of 79 characters, however,
+        we are not so strict - we let teams choose a maximum line length between 79 and 119 characters.
+    </p>
+    <p>2 blank lines should be left before functions and classes. Single blank lines are used to split sections and make logical breaks.</p>
+
+    <h3>Naming</h3>
+    <p>Module, file, function, and variable names (except type variables) should be lowercase and use underscores.</p>
+    <pre><code class="language-python"># File: my_module.py/mymodule.py
+
+def my_function():
+    my_variable = "value"</code></pre>
+    <p>Class and type variable names should use the camel case style.</p>
+    <pre><code class="language-python">from typing import List
+
+
+class MyClass:
+    pass
+
+ListOfMyClass = List[MyClass]</code></pre>
+    <p>Constant names should be all uppercase, and words should be separated with underscores.</p>
+    <pre><code class="language-python">MY_CONSTANT = 1</code></pre>
+    <p>
+        You should avoid single-character names, as these might be confusing.
+        But if you still do, you should avoid characters that may look like zero or one in some fonts:
+        "O" (uppercase o), "l" (lowercase L), and "I" (uppercase i).
+    </p>
+
+    <h3>Operators</h3>
+    <p>
+        If you have a chain of mathematic operations that you split into multiple lines,
+        you should put the operator at the beginning of the line and not to the end of the line.
+    </p>
+    <pre><code class="language-python"># No
+result = (
+    1 +
+    2 *
+    3
+)
+
+# Yes
+result = (
+    1
+    + 2
+    * 3
+)</code></pre>
+    <p>If you compare against <code>None</code>, you should use <code>is</code> and <code>is not</code>, but not compare equality.</p>
```

Comment:
```suggestion
    <p>If you ever check if something is equivalent to <code>None</code>, you should use <code>is</code> and <code>is not</code> instead of the <code>==</code> operator.</p>
```